x-pack/plugins/security_solution/common/cti/constants.ts

```diff
@@ -43,27 +43,27 @@ export enum ENRICHMENT_TYPES {
 }
 
 export const EVENT_ENRICHMENT_INDICATOR_FIELD_MAP = {
-  'file.hash.md5': 'threatintel.indicator.file.hash.md5',
-  'file.hash.sha1': 'threatintel.indicator.file.hash.sha1',
-  'file.hash.sha256': 'threatintel.indicator.file.hash.sha256',
-  'file.pe.imphash': 'threatintel.indicator.file.pe.imphash',
-  'file.elf.telfhash': 'threatintel.indicator.file.elf.telfhash',
-  'file.hash.ssdeep': 'threatintel.indicator.file.hash.ssdeep',
-  'source.ip': 'threatintel.indicator.ip',
-  'destination.ip': 'threatintel.indicator.ip',
-  'url.full': 'threatintel.indicator.url.full',
-  'registry.path': 'threatintel.indicator.registry.path',
+  'file.hash.md5': `${DEFAULT_INDICATOR_SOURCE_PATH}.file.hash.md5`,
+  'file.hash.sha1': `${DEFAULT_INDICATOR_SOURCE_PATH}.file.hash.sha1`,
+  'file.hash.sha256': `${DEFAULT_INDICATOR_SOURCE_PATH}.file.hash.sha256`,
+  'file.pe.imphash': `${DEFAULT_INDICATOR_SOURCE_PATH}.file.pe.imphash`,
+  'file.elf.telfhash': `${DEFAULT_INDICATOR_SOURCE_PATH}.file.elf.telfhash`,
+  'file.hash.ssdeep': `${DEFAULT_INDICATOR_SOURCE_PATH}.file.hash.ssdeep`,
+  'source.ip': `${DEFAULT_INDICATOR_SOURCE_PATH}.ip`,
+  'destination.ip': `${DEFAULT_INDICATOR_SOURCE_PATH}.ip`,
+  'url.full': `${DEFAULT_INDICATOR_SOURCE_PATH}.url.full`,
+  'registry.path': `${DEFAULT_INDICATOR_SOURCE_PATH}.registry.path`,
 };
 
 export const DEFAULT_EVENT_ENRICHMENT_FROM = 'now-30d';
 export const DEFAULT_EVENT_ENRICHMENT_TO = 'now';
 
 export const CTI_DATASET_KEY_MAP: { [key: string]: string } = {
-  'Abuse URL': 'threatintel.abuseurl',
-  'Abuse Malware': 'threatintel.abusemalware',
-  'AlienVault OTX': 'threatintel.otx',
-  Anomali: 'threatintel.anomali',
-  'Malware Bazaar': 'threatintel.malwarebazaar',
-  MISP: 'threatintel.misp',
-  'Recorded Future': 'threatintel.recordedfuture',
+  'Abuse URL': 'threat.abuseurl',
```

Comment:
Did you confirm that these dataset names are changing with the move to ECS 1.11? My expectation was that they would _not_ change, and that these `event.dataset` values would be the same in the upcoming integration packages. cc @P1llus

Reply:
@rylnd The event.dataset/event.module fields are based on the name of the package/module.

So these will not change for the filebeat modules themself, but they will change for packages yes.

Reply:
This is the mapping for one of the packages, following the package and dataset names:

```
- name: event.module
  type: constant_keyword
  description: Event module
  value: ti_cybersixgill
- name: event.dataset
  type: constant_keyword
  description: Event dataset
  value: ti_cybersixgill.threat
```